azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/AzureComputeProviderMetadata.java

```diff
@@ -71,8 +68,10 @@ public static Properties defaultProperties() {
       properties.put(CREDENTIAL_TYPE, CLIENT_CREDENTIALS_SECRET.toString());
       properties.put(RESOURCE_GROUP_NAME, "jcloudsgroup");
       properties.put(IMAGE_PUBLISHERS, "Canonical,RedHat");
-      properties.put(DEFAULT_IMAGE_LOGIN, "jclouds:Password1!");
       properties.put(TIMEOUT_NODE_TERMINATED, 60 * 10 * 1000);
+      // Default credentials for all images
+      properties.put("azurecompute-arm.image.login-user", "jclouds:Password1!");
+      properties.put("azurecompute-arm.image.authenticate-sudo", "true");
```

Comment:
BTW, these properties are read by [this class](https://github.com/jclouds/jclouds/blob/master/compute/src/main/java/org/jclouds/compute/config/GetLoginForProviderFromPropertiesAndStoreCredentialsOrReturnNull.java) which produces a LoginCredentials object that can be injected. [This class](https://github.com/jclouds/jclouds/blob/master/compute/src/main/java/org/jclouds/compute/strategy/impl/ReturnCredentialsBoundToImage.java) uses those default credentials to add them to all images. This is done [here](https://github.com/jclouds/jclouds/blob/master/compute/src/main/java/org/jclouds/compute/config/ComputeServiceAdapterContextModule.java#L121).